instrumentation/jaxws/jaxws-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxws/v2_0/WebServiceProviderSingletons.java

```diff
@@ -1,22 +0,0 @@
-/*
```

Comment:
this file should have been deleted in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5960